src/components/Modal/VaultStrategyModal/managedVaults/ManagedVaultsActivityTable.tsx

```diff
@@ -29,34 +29,35 @@ const ManagedVaultsActivityRow: FC = () => {
   }))
 
   return (
-    <div className="max-h-[350px] w-full overflow-y-scroll">
-      {activities.map((activity) => (
-        <>
-          <div className="grid grid-cols-[1fr,1fr,1fr] items-center border-b border-pink-800 p-3 text-xs font-normal max-md:grid-cols-[auto,130px,80px] max-md:gap-x-1 lg:gap-x-2 lg:px-6">
-            <span className="flex justify-center max-md:hidden">
-              {formatDate(activity.timestamp, "medium", "short")}
-            </span>
-            <span className="flex justify-center lg:hidden">
-              {formatDate(activity.timestamp, "short", "short")}
-            </span>
-            <span className="flex justify-center">
-              <Tooltip label="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam eu cursus lorem, et viverra mi. Cras ornare, ante in fringilla dignissim, ligula lectus finibus libero, in volutpat erat ipsum ut felis.">
-                <span>
-                  <div className="float-left mr-1 text-center">
-                    {activity.events}
-                  </div>
-                  <BsQuestionCircle className="float-left mt-[2px] h-4 w-4 cursor-pointer" />
-                </span>
-              </Tooltip>
-            </span>
-            <span className="flex justify-center max-md:hidden">
-              {shortenAddress(activity.explorer)}
-            </span>
-            <span className="flex justify-center lg:hidden">
-              {shortenAddress(activity.explorer, 2)}
-            </span>
-          </div>
-        </>
+    <div className="!mt-0 max-h-[600px] min-h-[300px] w-full overflow-y-scroll">
+      {activities.map((activity, i) => (
+        <div
+          key={`activity-${i}`}
+          className="grid grid-cols-[1fr,1fr,1fr] items-center border-b border-pink-800 p-3 text-xs font-normal max-md:grid-cols-[auto,130px,80px] max-md:gap-x-1 lg:gap-x-2 lg:px-6"
+        >
+          <span className="flex justify-center max-md:hidden">
+            {formatDate(activity.timestamp, "medium", "short")}
+          </span>
+          <span className="flex justify-center lg:hidden">
+            {formatDate(activity.timestamp, "short", "short")}
+          </span>
+          <span className="flex justify-center">
+            <Tooltip label="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam eu cursus lorem, et viverra mi. Cras ornare, ante in fringilla dignissim, ligula lectus finibus libero, in volutpat erat ipsum ut felis.">
+              <span>
+                <div className="float-left mr-1 text-center">
+                  {activity.events}
+                </div>
+                <BsQuestionCircle className="float-left mt-[2px] h-4 w-4 cursor-pointer" />
```

Comment:
`-0.5` is equivalent of `2px`; so `mt-0.5` is what you want here

Reply:
Also, using floats is kind of a dated technique. If it works, it works; but using flexbox would be better IMO.

Reply:
yes, with flex box we get the same results. Change it.